### DISCUSSION TITLE
Document Version 1.7

### DISCUSSION THREAD

**Discussion by rotkaepchen** [catfan/Medoo#961] [General]
Hello,
is there still a documentation for the version 1.7 available?
Currently I cannot use the version 2.0 because of an older php version.
Kind regards

**Comment by catfan** [comment]
There is no older documentation support for the v1.7.

You can check out the what's news https://medoo.in/api/whatsnew and upgrade guide https://medoo.in/api/upgrade to see the change. Otherwise, other things are still the same.

**Reply by rotkaepchen** [reply]
Thanks for the reply.

I have tried to use the version 2.0 with the same easy select command. 
But I got a failure :
 Call to undefined function Medoo\array_key_last()
May this happens because I use php version 7.2?

Thanks

**Reply by catfan** [reply]
@rotkaepchen The minimal version requirement of PHP is v7.3.

**Reply by rotkaepchen** [reply]
Thanks.
You can close this discussion :-)
